Flutter User Facing API.dart

```diff
@@ -0,0 +1,71 @@
+class Manager {
+  Future<Null> createClient(String restoreStateIdentifier) async
+  Future<Null> destroyClient() async
+
+  Future<Null> enableRadio()
+  Future<Null> disableRadio()
+  
+  Future<Null> cancelTransaction(String transactionId) async
+
+  Future<Null> setLogLevel(LogLevel logLevel)
+  Future<LogLevel> logLevel()
+
+  Future<BluetoothState> state()
+  Stream<BluetoothState> onStateChange()
+
+  Stream<ScanResult> startDeviceScan(int scanMode, int callbackType, List<String> uuids) async*
+  Future<Null> stopDeviceScan() async
+
+  Future<List<Device>> devices(String[] ids)
+  Future<List<Device>> connectedDevicesWithServices(String[] serviceUUIDs)
+}
+
+class Device {
+  Future<BleDevice> connect({ bool isAutoConnect, int requestMtu }) async
+  Future<bool> isConnected() async
+  Stream<BleDevice> onConnectionChanged()
+  Future<BleDevice> cancelConnection() async
+
+  Future<bool> requestMTU(int mtu, String transactionId) async
+  Future<int> readRSSI(String transactionId) async
+
+  Future<Null>  discoverAllServicesAndCharacteristics() async
+  Future<List<BleService>> services() async
+  Future<List<Characteristic>> characteristics(String serviceUUID) async
+  Future<List<Descriptor>> descriptors(String characteristicUUID) async
```

Comment:
What if more than one characteristic with given UUID is present? Is it currently the same on RN?